src/core_init_atmosphere/mpas_init_atm_static.F

```diff
@@ -281,8 +292,22 @@ subroutine init_atm_static(mesh, dims, configs)
           iPoint = nearest_cell(lat_pt,lon_pt,iPoint,nCells,maxEdges, &
                                 nEdgesOnCell,cellsOnCell, &
                                 latCell,lonCell)
-          ter(iPoint) = ter(iPoint) + rarray(i,j,1)
-          nhs(iPoint) = nhs(iPoint) + 1
+          if (bdyMaskCell(iPoint) < nBdyLayers) then
+             ter(iPoint) = ter(iPoint) + rarray(i,j,1)
+             nhs(iPoint) = nhs(iPoint) + 1
+          else
+             zPixel = sphere_radius * sin(lat_pt)
+             xPixel = sphere_radius * cos(lon_pt) * cos(lat_pt)
+             yPixel = sphere_radius * sin(lon_pt) * cos(lat_pt)
+
```

Comment:
I think it might be worth having a short comment here (and at further calls to `in_cell`) to explain this section. For instance:

```
! Avoid falsely inflating (or deflating) a regional mesh's border cells by determine if a pixel should be interpolated or ! not
```

Reply:
New comments have been added.